example_data/sample_bundle.json

```diff
@@ -268,7 +309,7 @@
               "coding": [
                 {
                   "system": "http://varnomen.hgvs.org",
-                  "code": "NM_001350626.1:c.1550G>T"
+                  "code": "c.1263del"
```

Comment:
this should also have the transcript in `NM_006...`, like it did before

Reply:
Sure